### ISSUE TITLE
Lack of newly raised optimizers

### ISSUE THREAD

**Issue by xsgxlz** [pytorch/pytorch#84347]
### 🚀 The feature, motivation and pitch

I'm working on train Sota DNNs for research purposes. Optimizers like LAMB, Ranger and Lookahead have shown their advantage compared to SGD or AdamW in many tasks, including large batch training. However, Pytorch has stopped adding new optimizers algorithms for a long time. There are some third-party optimizers packages though, but an official implementation will be helpful.

### Alternatives

Using third-party implementations

**Comment by yashika-git** [comment]
@albanD, Can I work towards adding Lookahead optimizer?

**Comment by xsgxlz** [comment]
> @albanD, Can I work towards adding Lookahead optimizer?

It could be help. From my view Lookahead is best to be a warpper for other optimizers rather than a separate one, to ensure flexibility.

**Comment by yashika-git** [comment]
@xsgxlz, Agreed. I'll try implementing it as a wrapper, not sure if I'll surely be able to do it, though.

**Comment by paulestano** [comment]
Hi. Is anybody working on this? 
I'm, in particular, interested in LARS-like optimizers (LARS, LAMB, Fromage, etc.). Is there any plan to merge any of these? and/or would it be worth to set up a pull request to contribute?
